Dockerfile

```diff
@@ -21,12 +21,12 @@
 ARG PY_VER=3.10-slim-bookworm
 
 # If BUILDPLATFORM is null, set it to 'amd64' (or leave as is otherwise).
-ARG BUILDPLATFORM=${BUILDPLATFORM:-amd64}
+ARG TARGET_BUILDPLATFORM=${BUILDPLATFORM:-amd64}
```

Comment:
would `ARG TARGET_BUILDPLATFORM=${TARGET_BUILDPLATFORM:-${BUILDPLATFORM:-amd64}}` be better? - haven't tested it?

I think it would provide a default if somehow both `TARGET_BUILDPLATFORM` and `BUILDPLATFORM` are not in context.

Reply:
Thanks @mistercrunch - I can confirm that it does work, although I'm not sure if it's necessary. `TARGET_BUILDPLATFORM` was just a variable I made up with the sole reason being that it's a different name to `BUILDPLATFORM`. Apologies if I've misunderstood though. Happy to make the change regardless

Reply:
oh why can't you use `BUILDPLATFORM`?

Reply:
From #31488 - podman gives this error:
![image](https://github.com/user-attachments/assets/260d4d78-53cf-4c3f-ba63-5166218d017d)